parts/k8s/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -1,72 +1,5 @@
 #!/bin/bash
-ERR_SYSTEMCTL_STOP_FAIL=3
```

Comment:
make sure you change the doc references that point to this (ie. troubleshooting doc, AKS wiki, etc)